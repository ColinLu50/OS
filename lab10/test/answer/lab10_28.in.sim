6150
657
607
496
599
492
