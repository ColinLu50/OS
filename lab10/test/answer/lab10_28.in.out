FCFS
95 -> 82 -> 446 -> 297 -> 342 -> 198 -> 47 -> 17 -> 244 -> 512 -> 189 -> 440 -> 276 -> 204 -> 209 -> 199 -> 444 -> 378 -> 358 -> 427 -> 217 -> 4 -> 290 -> 111 -> 420 -> 257 -> 235 -> 346 -> 83 -> 466 -> 240 -> 235 -> 300 -> 41 -> 356 -> 194 -> 91 -> 214 -> 351
6150
SSTF
95 -> 91 -> 83 -> 82 -> 111 -> 47 -> 41 -> 17 -> 4 -> 189 -> 194 -> 198 -> 199 -> 204 -> 209 -> 214 -> 217 -> 235 -> 235 -> 240 -> 244 -> 257 -> 276 -> 290 -> 297 -> 300 -> 342 -> 346 -> 351 -> 356 -> 358 -> 378 -> 420 -> 427 -> 440 -> 444 -> 446 -> 466 -> 512
657
SCAN
95 -> 91 -> 83 -> 82 -> 47 -> 41 -> 17 -> 4 -> 0 -> 111 -> 189 -> 194 -> 198 -> 199 -> 204 -> 209 -> 214 -> 217 -> 235 -> 240 -> 244 -> 257 -> 276 -> 290 -> 297 -> 300 -> 342 -> 346 -> 351 -> 356 -> 358 -> 378 -> 420 -> 427 -> 440 -> 444 -> 446 -> 466 -> 512
607
C-SCAN
95 -> 91 -> 83 -> 82 -> 47 -> 41 -> 17 -> 4 -> 0 -> 512 -> 512 -> 466 -> 446 -> 444 -> 440 -> 427 -> 420 -> 378 -> 358 -> 356 -> 351 -> 346 -> 342 -> 300 -> 297 -> 290 -> 276 -> 257 -> 244 -> 240 -> 235 -> 217 -> 214 -> 209 -> 204 -> 199 -> 198 -> 194 -> 189 -> 111
496
LOOK
95 -> 91 -> 83 -> 82 -> 47 -> 41 -> 17 -> 4 -> 111 -> 189 -> 194 -> 198 -> 199 -> 204 -> 209 -> 214 -> 217 -> 235 -> 240 -> 244 -> 257 -> 276 -> 290 -> 297 -> 300 -> 342 -> 346 -> 351 -> 356 -> 358 -> 378 -> 420 -> 427 -> 440 -> 444 -> 446 -> 466 -> 512
599
C-LOOK
95 -> 91 -> 83 -> 82 -> 47 -> 41 -> 17 -> 4 -> 512 -> 466 -> 446 -> 444 -> 440 -> 427 -> 420 -> 378 -> 358 -> 356 -> 351 -> 346 -> 342 -> 300 -> 297 -> 290 -> 276 -> 257 -> 244 -> 240 -> 235 -> 217 -> 214 -> 209 -> 204 -> 199 -> 198 -> 194 -> 189 -> 111
492
